korge-spine/src/commonTest/kotlin/com/esotericsoftware/spine/BonePositionTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.esotericsoftware.spine
+
+import com.esotericsoftware.spine.Animation.MixBlend
+import com.esotericsoftware.spine.Animation.MixDirection
+import com.esotericsoftware.spine.attachments.*
+import com.soywiz.korio.async.suspendTest
+import com.soywiz.korio.file.std.resourcesVfs
+import com.soywiz.korio.util.OS
+import kotlin.test.Test
+
+
+class BonePositionTest {
+    @Test
+    fun test() = suspendTest({ !OS.isJs }) {
+        val resource = resourcesVfs["spineboy/spineboy-ess.json"]
+        val json = SkeletonJson(AttachmentLoaderTest())
+        val skeletonData = json.readSkeletonData(resource)
+        val skeleton = Skeleton(skeletonData)
+        val bone = skeleton.findBone("gun-tip")
+        val fps = 1 / 15f
+        for (animation in skeletonData.animations) {
+            var time = 0f
+            while (time < animation.duration) {
+                animation.apply(skeleton, time, time, false, null, 1f, MixBlend.first, MixDirection.`in`)
+                skeleton.updateWorldTransform()
+                println(animation.name + "," + bone!!.worldX + "," + bone.worldY + "," + bone.worldRotationX)
```

Comment:
Instead of printing, could you emit the values and then check/assert them against a reference?

Reply:
This test was just a test imported from another spine runtime :)
Is it ok if I create a reference from one execution and then validate against that so we'll catch errors if things change ?

Reply:
Maybe we can stringify float/doubles by just taking one decimal into account. For example: `10.0.toStringDecimal(1)`
I guess that might prevent any rounding errors. Then you can make a single assert with a string with all the values already on it, so it is easier to change later.
You can also use:
```
"""
data...
""".trimIndent()
```
to store the values as a string, so it is easy to copy and paste if the assert changes. You can check a sample of this here:
https://github.com/korlibs/korge-next/blob/d757f79c057d0c2480fc665703cf2383ba011560/korge/src/jvmTest/kotlin/com/soywiz/korge/bitmapfont/BitmapFontTest.kt#L30-L68

Unless the output is going to be stable, I usually like to have as few tests like that as possible. Maybe one per entity.
But here I guess that if the matrix output is consistent, this bone test should be stable enough.